test/integration_tests/test_reuse.py

```diff
@@ -44,6 +45,7 @@ def tearDown(self):
         execute_query_fetch_all("DROP TABLE IF EXISTS DETRAC;")
 
     def _verify_reuse_correctness(self, query, reuse_batch):
+        gc.collect()
```

Comment:
Should we add a comment here for why this is needed? Links to relevant pages would also be helpful.